lib/feeder_importer.rb

```diff
@@ -0,0 +1,72 @@
+# import script for a podcast live in feeder & dovetail
+require 'prx_access'
+
+class FeederImporter
```

Comment:
So this is pretty much a standalone script at the moment, right?  (Not accessible via the api)